devices/Mpu6886/README.md

```diff
@@ -0,0 +1,128 @@
+```
+
+See [samples](samples) for a complete sample application.
+
+## Calibration
+
+The gyroscope can be calibrated using the `Calibrate` function. With the `iterations` parameter
+you can specify how many values should be read from the sensor to calculate an average (1000 iterations seems
+to be a good number). During the calibration you want to keep the sensor still and in a fixed position (e.g. lying
+flat on a table). The `Calibrate` function will write the values to the corresponding compensation registers
+of the sensor, values are corrected when retrieved automatically.
+
+The return value of the `Calibrate` method gives a vector containing the 3 (for the X,Y and Z axes) calculated
+compensation values.
+
+```csharp
+var offset = ag.Calibrate(1000);
+Debug.WriteLine($"Calibration done, calculated offsets {offset.X} {offset.Y} {offset.Y}");
+```
+
+It is possible to write directly to the compensation registers using the `SetGyroscopeOffset` function. You might
+want to do this when you create your own custom calibration method, or you retrieve existing calibration data from
+a persisted data store or memory location.
+
+⚠ **When the devices reboots, the offset registers are cleared.** So if you don't persist the calibration data yourself
+you will have to run the calibration method every time the device boots.
+
+## Sleep mode
+
+The sensor can be put in sleep mode by calling the `Sleep` function. After that the `WakeUp` function should be called.
+
+## Setting scales
+
+Both for the gyroscope and accelerometer you can set the desired scales using the `AccelerometerScale` and `GyroscopeScale`
+properties.
+
+```csharp
+// Set the scale of the accelerometer to 2G
+ag.AccelerometerScale = AccelerometerScale.Scale2G;
+```
+
+## Setting enabled axes
+
+By default all gyroscope and accelerometer axes are enabled. If desired you can specify explicitly which axes should be enabled.
+In that case, all other axes will be disabled. When an axis is disabled, data can still be read for that axis but the values will not
+change anymore.
+
+```csharp
+// Enable only the X axis of the gyroscope and accelerometer
+ag.EnabledAxes = EnabledAxis.GyroscopeX | EnabledAxis.AccelerometerX;
+```
+
+## Features not implemented
+
+The following features are enabled in the MPU6886 but not yet implemented in this driver:
+
+- registers `0x05` to `0x0b`: low noise to low power offset
+- registers `0x0d` to `0x0F`: self test
+- register `0x1a`: FIFO mode, DLPF (digital low pass filter)
+- register `0x1d` bit 0 to 3: DLPF
+- register `0x1e` bit 4 to 6: averaging filter for low power mode
+- registers `0x20` to `0x22`: threshold for wake-on motion
+- register `0x36`: fsync interrupt status
+- register `0x37`: INT/DRDY pin config
+- register `0x39` and `0x3a`: FIFO watermark interrupt
+- register `0x60` and `0x61`: FIFO watermark interrupt
+- register `0x68`: signal path reset
+- register `0x69`: acc. intelligence control
```

Comment:
missing an extra enter at the end (to be Markdown perfect ;-))

Reply:
Already there, so leaving this one :)